src/modules/villages/village_handler.ts

```diff
@@ -11,4 +12,17 @@ router.get(
     res.status(httpStatus.OK).json(await Service.findAllWithLocation(req.query))
   })
 
+router.get(
+  '/v1/villages/:id',
+  async (req: Request<never, never, never, Entity.RequestQuery>, res: Response, next: NextFunction) => {
```

Comment:
```suggestion
  async (req: Request, res: Response, next: NextFunction) => {
```

please remove if not use

Reply:
turns out I need the params, I will change it